shell/components/fleet/dashboard/Empty.vue

```diff
@@ -0,0 +1,77 @@
+
+<script>
+import { FLEET } from '@shell/config/types';
+import { NAME } from '@shell/config/product/fleet';
+export default {
+  name: 'FleetDashboardEmpty',
+
+  props: {
+    permissions: {
+      type:     Object,
+      required: true
+    },
+  },
+
+  data() {
+    return {
+      getStartedLink: {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          product:  NAME,
+          resource: FLEET.GIT_REPO
+        },
+      },
+    };
+  }
+};
+</script>
+<template>
+  <div
+    class="fleet-empty-dashboard"
+    :data-testid="'fleet-empty-dashboard'"
+  >
+    <i class="icon-fleet mb-30" />
+    <h1>{{ t('fleet.dashboard.welcome') }}</h1>
+    <p class="mb-30">
+      <span>{{ t('fleet.dashboard.gitOpsScale') }}</span>
+      <a
+        :href="t('fleet.dashboard.learnMoreLink')"
+        target="_blank"
+        rel="noopener noreferrer nofollow"
+      >
+        {{ t('fleet.dashboard.learnMore') }} <i class="icon icon-external-link" />
+      </a>
```

Comment:
I think that we should add some hidden text so that screen readers can announce that the link will open in a new tab. Otherwise, opening a new tab can be disorienting to users who rely on assistive tech. 

We can do something like this:

```html
  <span class="sr-only">{{ t('generic.opensInNewTab') }}</span>
```

FYI @aalves08